include/geom/face_quad4.h

```diff
@@ -159,6 +159,27 @@ class Quad4 : public Quad
 
 };
 
+class QuadShell4 : public Quad4
```

Comment:
It's more an organizational thing than a number of new lines of code per new file thing.  There is one header per geometric element type at the moment, I'd like to keep it that way...